cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py

```diff
@@ -42,8 +55,7 @@ def test_no_options(self):
             _make_task(GenerateDataFromYaml, {})
 
     def test_simple(self):
-        with NamedTemporaryFile() as t:
-            database_url = f"sqlite:///{t.name}"
+        with temp_sqlite_database_url() as database_url:
```

Comment:
Personally I probably would have set up database_url as a pytest fixture (though that does require switching away from unittest.TestCase)